doc/guides/from_nest2_to_nest3.rst

```diff
@@ -1248,9 +1250,92 @@ probability and delay, and random weights from a normal distribution:
+
+With NEST 3.0 we replaced this single class by an extensible and
+modular infrastructure for handling recordings: each modality is now
+taken care of by a specific recording backend and each recorder can
+use one of them to handle its data.
+
+NEST 3.0 brings recording backends for all modalities supported
+already in NEST 2.x. If compiled with support for `SIONlib
+<http://www.fz-juelich.de/jsc/sionlib>`_, an additional backend for
+writing binary files in parallel becomes available. This is especially
+useful on large clusters and supercomputers.
+
+Changes
+^^^^^^^
+
+In NEST 2.x, the recording modality was selected by either providing a
+list of modalities to the `record_to` property, or by setting one or
+more of the flags `to_file`, `to_memory`, or `to_screen` to *True*.
+
+In NEST 3.0, the individual flags are gone and the `record_to`
+property now expects the name of the backend to use. Recording to
+multiple modalities from a single device is not possible anymore and
+individual devices have to be created and configured if this
+functionality is needed.
+
+All following examples assume that the variable `mm` points to a
+``multimeter`` instance, i.e. that ``mm = nest.Create('multimeter')``
+was executed.
+
+
+  +------------------------------------------------------+--------------------------------+
+  | NEST 2.x                                             | NEST 3.0                       |
+  +------------------------------------------------------+--------------------------------+
+  |                                                      |                                |
+  | ::                                                   | ::                             |
+  |                                                      |                                |
+  |     nest.SetStatus(mm, {'record_to': ["file"]})      |     mm.record_to = "ascii"     |
+  |     nest.SetStatus(mm, {'record_to': ["screen"]})    |     mm.record_to = "screen"    |
+  |     nest.SetStatus(mm, {'record_to': ["memory"]})    |     mm.record_to = "memory"    |
+  |                                                      |                                |
+  +------------------------------------------------------+--------------------------------+
+  | ::                                                   | ::                             |
+  |                                                      |                                |
+  |     nest.SetStatus(mm, {'to_file': True})            |     mm.record_to = "ascii"     |
+  |     nest.SetStatus(mm, {'to_screen': True})          |     mm.record_to = "screen"    |
+  |     nest.SetStatus(mm, {'to_memory': True})          |     mm.record_to = "memory"    |
+  |                                                      |                                |
+  +------------------------------------------------------+--------------------------------+
+
+The list of available backends can be retrieved using the following command:
```

Comment:
```suggestion
You can retrieve the list of available backends using the following command:
```
Nothing wrong with what you had, but I tend to prefer including some more active voice in the instructions